common/src/main/scala/uk/ac/wellcome/models/MergedSierraRecord.scala

```diff
@@ -23,15 +23,15 @@ case class MergedSierraRecord(
   id: String,
   maybeBibData: Option[SierraBibRecord] = None,
   itemData: Map[String, SierraItemRecord] = Map[String, SierraItemRecord](),
-  version: Int = 1
+  version: Int = 0
 ) extends Transformable {
 
   /** Given a new bib record, construct the new merged row that we should
     * insert into the merged database.
     *
     * Returns None if there's nothing to do.
```

Comment:
Not right now?

Reply:
I think conceptually this should have been "construct a list of merge operations to perform" ... I think that was the intention of returning an option in any case.